### PR TITLE
Use `value > max` style in pylint and mccabe messages

### PR DESCRIPTION
**Pull request by edgarrmondragon**
Closes https://github.com/charliermarsh/ruff/issues/3539

### CODE REVIEW

File: crates/ruff/src/rules/mccabe/rules.rs [hunk 1]
```diff
@@ -50,13 +50,18 @@ use ruff_python_ast::source_code::Locator;
...
     #[derive_message_formats]
     fn message(&self) -> String {
-        let ComplexStructure { name, complexity } = self;
-        format!("`{name}` is too complex ({complexity})")
+        let ComplexStructure {
+            name,
+            complexity,
+            max_complexity,
+        } = self;
+        format!("`{name}` is too complex ({complexity} > {max_complexity})")
```
Comment (charliermarsh) [inline crates/ruff/src/rules/mccabe/rules.rs:64]: Do you mind changing those Pylint messages too, like `too_many_branches`, to use this style?

Comment (edgarrmondragon) [reply crates/ruff/src/rules/mccabe/rules.rs:64]: Sure 👍

Comment (edgarrmondragon) [reply crates/ruff/src/rules/mccabe/rules.rs:64]: Done (e17a361) and updated the title too

**Comment by charliermarsh** [comment]
Thank you!

**Pull request merged**